frontend/src/component/project/Project/ProjectOverview.tsx

```diff
@@ -108,6 +109,29 @@ const NewProjectOverview: FC<{
         refreshInterval,
     });
 
+    const { openFeedback, hasSubmittedFeedback } = useFeedback(
```

Comment:
I'd extract all this code into one custom hook e.g. useRefactoredProjectOverviewFeedback. Currently it's very distracting with all the low level details